core/templates/dev/head/components/ParamChangesEditorDirective.js

```diff
@@ -202,7 +202,9 @@ oppia.directive('paramChangesEditor', [function() {
 
           explorationParamSpecsService.saveDisplayedValue();
           $scope.paramChangesService.saveDisplayedValue();
-          $scope.postSaveHook();
+          if ($scope.postSaveHook) {
```

Comment:
Yes. It gets set in the settings tab of the editor (for the exploration-level changes).